src/templates/next/components/InfoCards/InfoCards.tsx

```diff
@@ -0,0 +1,116 @@
+import { InfoCardsProps } from "~/common"
+import Card from "../shared/Card"
+
+const TitleSection = ({
+  title,
+  subtitle,
+  className,
+}: {
+  title: InfoCardsProps["title"]
+  subtitle: InfoCardsProps["subtitle"]
+  className?: string
+}) => {
+  return (
+    <div className={`flex flex-col gap-8 self-start max-w-3xl ${className}`}>
+      <h3 className="text-content text-2xl leading-tight font-semibold sm:text-4xl ">
```

Comment:
seems like should be `text-content-strong`
also could consider using `/typography/Heading` which follows the typography tokens in design exactly (font size, line height, weight, letter spacing)

although might need to refactor it to expose `lg` and `sm` because I encoded the responsiveness in while it seems like the smaller screen designs here still use `heading-03-lg` 😢

Reply:
Makes sense, have changed it in https://github.com/isomerpages/isomer-components/pull/38/commits/b1c5b322078659e1cb0b17bfe57302f4c9f1e3e6. The small screen designs do use `heading-03-sm`! the medium ones are using lg which is fine